libvast/vast/defaults.hpp

```diff
@@ -149,6 +149,25 @@ struct pcap {
 
 } // namespace import
 
+// -- constants for the explore command and its subcommands --------------------
+
+namespace explore {
+
+// A value of zero means 'unlimited' for all three limits below.
+// If all limits are non-zero, the number of results is bounded
+// by `min(max_events, limit1*limit2)`.
+
+/// Maximum total number of results.
+constexpr size_t max_events = 0;
+
+/// Maximum number of results for the initial query.
+constexpr size_t limit1 = 1000;
+
+/// Maximum number of results for every explored context.
+constexpr size_t limit2 = 1000;
+
+} // namespace explore
```

Comment:
These numbers are basically just guesses, not sure if there's a more systematic way to determine good defaults.

Reply:
If I recall correctly we've used 100 for other default max event limits before.